mondey_backend/src/mondey_backend/import_data/manager/data_manager.py

```diff
@@ -0,0 +1,262 @@
+
+    labels_path: Path
+    data_path: Path
+    questions_configured_path: Path
+    milestones_metadata_path: Path
+    additional_data_path: Path | None = None
+
+    @classmethod
+    def from_strings(
+        cls,
+        labels_path: str,
+        data_path: str,
+        questions_configured_path: str,
+        milestones_metadata_path: str,
+        additional_data_path: str | None = None,
+    ) -> ImportPaths:
+        """Create ImportPaths from string paths."""
+        return cls(
+            labels_path=Path(labels_path),
+            data_path=Path(data_path),
+            questions_configured_path=Path(questions_configured_path),
+            milestones_metadata_path=Path(milestones_metadata_path),
+            additional_data_path=Path(additional_data_path)
+            if additional_data_path
+            else None,
+        )
+
+    @classmethod
+    def default(cls, base_dir: Path | None = None) -> ImportPaths:
+        """Create default ImportPaths."""
+        import_dir = app_settings.PRIVATE_FILES_PATH
+
+        return cls(
+            labels_path=import_dir / "labels_encoded.csv",
+            data_path=import_dir / "data.csv",
+            questions_configured_path=import_dir / "questions_specified.csv",
+            milestones_metadata_path=import_dir / "milestones_metadata_(variables).csv",
+            additional_data_path=import_dir / "additional_data.csv",
+        )
+
+
+class DataManager:
+    """
+    Manager for data loading and database session handling.
+
+    This class handles:
+    1. Loading and parsing CSV data
+    2. Creating and managing database connections
+    """
+
+    def __init__(
+        self,
+        import_paths: ImportPaths | None = None,
+        mondey_db_path: Path | None = None,
+        current_db_path: Path | None = None,
+        users_db_path: Path | None = None,
+        debug: bool = False,
+    ):
+        """
+        Initialize the DataManager.
+
+        Args:
+            import_paths: Paths to import data files
+            mondey_db_path: Path to the Mondey database
+            current_db_path: Path to the current database
+            users_db_path: Path to the users database
+            debug: Enable debug logging
+        """
+        self.debug = debug
+        self._setup_logging()
+
+        # Set up paths
+        script_dir = Path(__file__).parent.parent.parent.parent.parent.absolute()
+        self.import_paths = import_paths or ImportPaths.default(script_dir)
+
+        # Database paths
+        self.current_db_path = current_db_path or (
+            script_dir / "src/mondey_backend/import_data/current_db/current_mondey.db"
+        )
+        self.users_db_path = users_db_path or (
+            script_dir / "src/mondey_backend/import_data/current_db/current_users.db"
+        )
+
+        # Database connections
+        # When we want this to work directly on the live DB, we can change these paths to be the same as the normal ones.
```

Comment:
when using the live db the endpoint can just pass the `session` and `user_session` deps directly - then we don't need to create the engines or the sessions here